File: gl main/corejava/src/corejava/Array2.java
package corejava;

public class Array2 {
	
	void disp(String name,int y,int... ar){
		System.out.println(name);
		for (int i : ar) {
			System.out.print(i+"  ");
		}
		System.out.println(ar[0]);
		
	}
	
	public static void main(String... args) {
		
		Array2 array2=new Array2();
		array2.disp("ram kumar", 11,22,33,44,55);
		int ar[]={1,2,3,4};
		
		array2.disp("ram kumar", 22,ar);
	}
	
}

File: coforge_sep_2024-main/nodejs/array2.js
var arr=[11,2,3,44,55,6,78]

function sum(a){
    let s=0;
     for (const element of a) {
        s=s+element;
     }
     return s;
}

console.log("sum of "+arr+" is "+sum(arr))

var name=["ram kumar","amit kumar","jayant kumar"]
console.log("unsorted "+name)
console.log("sorted "+name.sort())


function myfunction(a,b){
    return a-b
}
arr.sort(myfunction)
console.log('sorted array'+arr)

